.circleci/deploy_and_build_docker.py

```diff
@@ -0,0 +1,14 @@
+import sys
+from utils import build_and_push_docker
+
+arg_dir = sys.argv[1]
+arg_container = sys.argv[2]
+arg_tag = sys.argv[3]
+
+
+def main(dir, container, tag):
+  return build_and_push_docker(dir, container, tag)
```

Comment:
`dir` is frowned upon as a variable in Python, because it's the name of a [built-in function](https://docs.python.org/3.5/library/functions.html#built-in-funcs). Shadowing built-in functions/constants can have unexpectedly weird side effects. `dirname`?